doc/source/conf.py

```diff
@@ -223,7 +223,7 @@ def cd(newpath):
 
 # For trac custom roles
 
-default_role = 'math'
+default_role = 'obj'
```

Comment:
Just to clarify, how will `default_role = 'obj'` behave differently from the default setting, which is `default_role = None`? The [Sphinx documentation][2] states that the `:py:obj:` role is for Python objects of unspecified type. What will happen if the function docstring:

    def foo(degree):
        """Do something with `degree`."""

In this case, `degree` is a function (object) in the `networkx` package, but the intention in this docstring is to render `degree` as a variable name (e.g. in italics).

[1]: http://www.sphinx-doc.org/en/stable/config.html#confval-default_role
[2]: http://www.sphinx-doc.org/en/stable/domains.html#role-py:obj